test/index.js

```diff
@@ -1,2 +1,3 @@
+require('any-promise/register')('bluebird', {Promise: require('bluebird')});
```

Comment:
do we need a test without registering a promise library, or would that just be writing tests for any-promise?